artemis/src/main/java/tech/pegasys/artemis/Artemis.java

```diff
@@ -27,18 +27,24 @@ public static void main(final String... args) {
       CommandLineArguments cliArgs = new CommandLineArguments();
       CommandLine commandLine = new CommandLine(cliArgs);
       commandLine.parse(args);
-      BeaconNode node = new BeaconNode(commandLine, cliArgs);
-      node.start();
-      // Detect SIGTERM
-      Runtime.getRuntime()
-          .addShutdownHook(
-              new Thread() {
-                @Override
-                public void run() {
-                  System.out.println("Artemis is shutting down");
-                  node.stop();
-                }
-              });
+      if (commandLine.isUsageHelpRequested()) {
+        commandLine.usage(System.out);
+        return;
+      } else {
+
+        BeaconNode node = new BeaconNode(commandLine, cliArgs);
+        node.start();
+        // Detect SIGTERM
+        Runtime.getRuntime()
+            .addShutdownHook(
+                new Thread() {
+                  @Override
+                  public void run() {
```

Comment:
FYI this will hang on a halt when running a debugging session

Reply:
Can u open an issue?